recipes/graspologic-native/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "graspologic-native" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/graspologic_native-{{ version }}.tar.gz
+  sha256: 72b7586028a91e9fef9af0ef314d368f0240c18dca99e6e6c546334359a8610a
+
+build:
+  script:
+    - cargo bundle-licenses --format yaml --output THIRDPARTY.yml
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - cargo-bundle-licenses
+    - python >=3.6,<3.12                     # [build_platform != target_platform]
```

This should not be pinned.
```suggestion
    - python                     # [build_platform != target_platform]
```